projects/js-packages/licensing/components/activation-screen-success-info/index.jsx

```diff
@@ -24,23 +23,19 @@ import './style.scss';
  * @param {object} props -- The properties.
  * @param {number} props.productId -- The id of the product activated
  * @param {string} props.siteRawUrl -- The url of the site
+ * @param {string} props.initialStateRecommendationsStep -- The current recommendation step.
```

Comment:
I think we should rename `initialStateRecommendationsStep` to `currentRecommendationsStep`.

Reply:
Thanks! I agree the name you suggested is way better. Fixed in: [de5c6d](https://github.com/Automattic/jetpack/pull/22207/commits/de5c6d6bfb811465b4e942e9e76d417bbea8b934)